File: android/src/main/java/me/carda/awesome_notifications/awesome_notifications_android_core/listeners/AwesomeNotificationEventListener.java
package me.carda.awesome_notifications.awesome_notifications_android_core.listeners;

import me.carda.awesome_notifications.awesome_notifications_android_core.models.returnedData.NotificationReceived;

public interface AwesomeNotificationEventListener {
    public void onNewNotificationReceived(String eventName, NotificationReceived notificationReceived);
}

File: ios/Classes/lib/core/listeners/AwesomeNotificationEventListener.swift
//
//  AwesomeNotificationEventListener.swift
//  awesome_notifications
//
//  Created by CardaDev on 30/01/22.
//

import Foundation

public protocol AwesomeNotificationEventListener: AnyObject {
    func onNewNotificationReceived(eventName:String, notificationReceived:NotificationReceived);
}
